src/Symfony/Component/Routing/Exception/MissingMandatoryParametersException.php

```diff
@@ -19,4 +19,36 @@
  */
 class MissingMandatoryParametersException extends \InvalidArgumentException implements ExceptionInterface
 {
+    private array $missingParameters = [];
+    private string $routeName = '';
+
+    public function __construct(/*array*/ $missingParameters = [], /*string*/ $routeName = '', /*int*/ $code = 0, \Throwable $previous = null)
+    {
+        if (\is_array($missingParameters)) {
+            $this->missingParameters = $missingParameters;
+            $this->routeName = $routeName;
+            $message = sprintf(
+                'Some mandatory parameters are missing ("%s") to generate a URL for route "%s".',
```

Comment:
shouldn't this say `for the route` ? /cc @weaverryan

Reply:
I  think you may be right, as in  `... for the route named "%s"`. I'll wait a bit for a native speaker answer ^^